application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCommand.java

```diff
@@ -0,0 +1,145 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.unions.MessageChannelUnion;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+
+import java.util.Objects;
+
+public final class BookmarksCommand extends SlashCommandAdapter {
+
+    private static final Logger logger = LoggerFactory.getLogger(BookmarksCommand.class);
+
+    public static final String COMMAND_NAME = "bookmarks";
+    public static final String SUBCOMMAND_ADD = "add";
+    public static final String SUBCOMMAND_LIST = "list";
+    public static final String SUBCOMMAND_REMOVE = "remove";
+    public static final String ADD_OPTION_NOTE = "note";
```

Comment:
id go for just `NOTE_OPTION` or `BOOKMARK_NOTE_OPTION`.

Reply:
But then it it not really clear that the note option is used for the add subcommand

Reply:
`ADD_BOOKMARK_NOTE_OPTION` maybe?